k8s/helmfile/env/staging/argo-cd-base.values.yaml.gotmpl

```diff
@@ -8,10 +8,18 @@ dex:
 controller:
   resources:
    limits:
-     memory: 1024Mi
+     memory: 512Mi
    requests:
      cpu: 250m
-     memory: 1024Mi
+     memory: 256Mi
```

Comment:
This is now actually lower than the average usage. May not be an issue but something to keep an eye on if this ends up regularly being evicted at time of high memory usage across the whole cluster

Reply:
Ah interesting, you're right! I just realized the staging requests were much higher than in prod and copied those. But yeah, on prod the usage is also above average, guess I'll adjust it then 
![image](https://github.com/user-attachments/assets/f110a0f5-50d7-427a-ac86-dd943e834b4b)